metron-analytics/metron-profiler/README.md

```diff
@@ -69,26 +69,27 @@ More information on configuring and using the client can be found [here](../metr
 It is assumed that the `PROFILE_GET` client is correctly configured before using it.
     ```
     $ bin/stellar -z node1:2181
-    
+
```

Comment:
Removing the indenting isn't sufficient to prevent the mis-formatting.  The whole blank line needs to be removed.